testutils/infra/cli.py

```diff
@@ -144,12 +161,12 @@ def add_default_tenant_token(self, tenant_token):
         :param tenant_token - 'the default tenant token to set'
         """
 
-        # Append the default_tenant_token in the config ('/etc/deviceauth/config.yaml')
+        # Append the default_tenant_token in the config (f'{self.service.data_path}/config.yaml')
```

Comment:
Maybe the comment does not need the path explicitly, it is four lines below anyway...